.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
```

Comment:
- [ ] Good job adding the node modules to `.gitignore` 👍🏽. I can see that the node_modules is still being pushed to GitHub because GitHub is already tracking it. To un track, enter `git rm -r --cached node_modules` in your terminal so that node_modules will no longer be pushed to git to meet the project requirement.